torch/nn/attention/flex_attention.py

```diff
@@ -751,9 +770,15 @@ def score_mod(
         key (Tensor): Key tensor; shape :math:`(B, H, S, E)`.
         value (Tensor): Value tensor; shape :math:`(B, H, S, Ev)`.
         score_mod (Optional[Callable]): Function to modify attention scores. By default no score_mod is applied.
-        block_mask (BlockMask): BlockMask object that controls the blocksparsity pattern of the attention.
-        scale (Optional[float]): Scaling factor applied prior to softmax. If
+        block_mask (Optional[BlockMask]): BlockMask object that controls the blocksparsity pattern of the attention.
+
+    Kwargs:
+        scale (float): Scaling factor applied prior to softmax. If
         none, the default value is set to :math`\frac{1}{\sqrt{E}}`
+        rows_guaranteed_safe (bool): For all rows, is it guaranteed into a (BLOCK_M, BLOCK_N) block that at least one value
```

Comment:
I think we should leave them undocumented. For now, i'd consider these an "undocumented feature only"